net/dnsdist/Makefile

```diff
@@ -0,0 +1,86 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=dnsdist
+PKG_VERSION:=1.3.3
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+PKG_SOURCE_URL:=https://downloads.powerdns.com/releases/
+PKG_HASH:=9fb24f9032025955169f3c6e9b0a05b6aa9d6441ec47da08d22de1c1aa23e8cf
+
+PKG_MAINTAINER:=James Taylor <james@jtaylor.id.au>
+PKG_LICENCE:=GPL-2.0-only
+PKG_LICENCE_FILES:=COPYING
+
+PKG_INSTALL:=1
+
+PKG_CONFIG_DEPENDS:= \
+  CONFIG_DNSDIST_GNUTLS \
+  CONFIG_DNSDIST_OPENSSL
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/dnsdist/config
+comment "SSL support"
+
+choice
+        prompt "Selected SSL library"
+        default DNSDIST_OPENSSSL
+
+        config DNSDIST_OPENSSL
+                bool "OpenSSL"
+
+        config DNSDIST_GNUTLS
+                bool "GnuTLS"
+
+        config DNSDIST_NOSSL
+                bool "No SSL support"
+
+endchoice
+endef
+
+define Package/dnsdist
+  SECTION:=base
+  CATEGORY:=Network
+  TITLE:=dnsdist DNS-, DOS- and abuse-aware loadbalancer.
+  DEPENDS:=+DNSDIST_OPENSSL:libopenssl +DNSDIST_GNUTLS:libgnutls +protobuf +re2 +libedit +libfstrm +libsodium +lua +boost +libnetsnmp
+  URL:=http://dnsdist.org/
+endef
+
+define Package/dnsdist/description
+  dnsdist is a highly DNS-, DoS- and abuse-aware loadbalancer. Its goal in life
+  is to route traffic to the best server, delivering top performance to legitimate
+  users while shunting or blocking abusive traffic.
+endef
+
+define Package/dnsdist/conffiles
+/etc/dnsdist/dnsdist.conff
+/etc/config/dnsdist
+/etc/init.d/dnsdist
+endef
+
+CONFIGURE_ARGS+= \
+	--prefix=/usr \
+	--sysconfdir=/etc \
+	--enable-dnscrypt \
+	$(if $(CONFIG_DNSDIST_NOSSL),,--enable-dns-over-tls) \
```

Comment:
I might be reading this wrong, but shouldn't this be --disable?

Reply:
This feature is disabled by default so I just added in a blank ',,' so it wouldn't be enabled when no SSL is available, and would be enabled if any are compiled. I can make it explicit if that would help

Reply:
Ah I misread it.